### PR TITLE
Add API method for downloading aggregated logs

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: cmd/wstail/tail.go [hunk 1]
```diff
@@ -273,6 +275,26 @@ func serveWs(w http.ResponseWriter, r *http.Request) (err error) {
 	return nil
 }
 
+// downloadLogs serves /v1/download
+//
+// it compresses the file with aggregated logs and pushes it to the response
+// stream
+func downloadLogs(w http.ResponseWriter, r *http.Request) error {
+	file, err := os.Open(filePath)
```
Comment (a-palchikov) [inline cmd/wstail/tail.go]: Do you want to grab the rotated messages as well or just the active one?

Comment (r0mant) [reply cmd/wstail/tail.go]: I guess we could include older logs too. Would they be named just like `filePath+.<number>` (e.g. `/var/log/messages.1, /var/log/messages.2`) or they are named differently?

**Review by a-palchikov — COMMENTED** [review]
One question regarding the need to archive rotated messages - otherwise lgtm

File: cmd/wstail/tail.go (hunk 1 above)
Comment (a-palchikov) [reply cmd/wstail/tail.go]: yeah, I think it's about right - I'm not 100% sure, but it should be standard (`savelog`)

**Comment by r0mant** [comment]
@a-palchikov I have updated the PR to collect all logs. You might wanna take another look cause the implementation changed significantly :)

**Comment by r0mant** [comment]
Anyway, I've tested it and it works cool so I'm gonna merge.

**Pull request merged**